src/OpenLoco/Vehicles/Vehicle.h

```diff
@@ -193,7 +193,7 @@ namespace OpenLoco::Vehicles
         static constexpr auto vehicleThingType = VehicleThingType::head;
         uint8_t pad_20;
         company_id_t owner; // 0x21
-        uint16_t var_22;
+        uint16_t name;      // 0x22
```

Comment:
Make the type string id and add a note that the ordinal number is to be combined with this as an argument